packages/vscode-extension/src/debug/commonUtils.ts

```diff
@@ -38,6 +38,24 @@ export async function getLocalEnv(): Promise<{ [key: string]: string } | undefin
   if (isMultiEnvEnabled()) {
     // use localSettings.json as input to generate the local debug envs
     env = await getLocalDebugEnvs();
+
+    // try load .env.teamsfx.local for tab/func/bot
```

Comment:
does CLI need this fix?

Reply:
Good catch. I'll move this to local debug plugin.

Reply:
Moved changes to local debug plugin so it works for CLI as well. Added test case.